source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+    Basic mobile device requirements are provided in the link above.
+
+`Network Requirements <https://>`_
+
+********************************
+Public App Store vs. Custom Apps
+********************************
+
+The most critical decision you will make in your mobile deployment will be whether to use the apps provided by Mattermost via the public App Stores (i.e. Google Play And Apple App Store), or to build and distribute custom versions of the Mattermost apps. Below we provide a very general overview of these options.
+
+Public App Store
+================
+
+Utilizing the public app stores greatly reduces the deployment time and is our recommended approach. Key benefits include… 
+
+- Easier deployment, driven by user needs.
+- Ability to use the Mattermost Push Notification Service, or MPNS.
+- Apps automatically updated with the latest features, enhancements and security updates.
+
+Again, this is the recommended route which is as simple as downloading the Mattermost mobile application from the corresponding store, entering a URL to point towards your Mattermost server, and logging in.
+
+Custom Apps
+===========
+
+If you desire to customize the applications, or do not want your users downloading the application from the public app stores, you will need to build the apps yourself. 
+
+As the apps are an open source project, it will require a fork, and your team will be responsible for maintaining the fork, as well as keeping the fork up to date with any changes made by Mattermost.
+
+This process can be complicated. It will also greatly increase deployment time, not only initially, but whenever the apps need to be updated. 
+
+We recommend having your development team take a look at the documentation to ensure they understand the scale and requirements of this path. In general, this route will present some challenges, including…
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
+
+*******************************************
+Mattermost Push Notification Service (MPNS)
+*******************************************
+
+Receiving notifications on a mobile device is a core value of your mobile deployment.
+
+It also represents a return on your deployment investment through better-connected users. 
+
+A push proxy is a key technology behind notification transmission.  It enables notifications between the server and the mobile application.
+
+Mattermost provides a self-hosted Push Proxy you can deploy, the Mattermost Push Notification Service (MPNS). This is also available via a hosted-by-mattermost option, or hosted push notification service (HPNS)*.
+
+    **Note:** `Only Mattermost Enterprise Edition E10 <https://about.mattermost.com/pricing/>`_ or higher to receive access to our Hosted Push Notification Service (HPNS)
```

Comment:
Team edition can use TPNS

Reply:
+1

Reply:
should this be included in a guide focused on enterprise? 

from the [docs](https://docs.mattermost.com/mobile/mobile-hpns.html): 'It does not offer a production-level uptime service level agreements (SLAs).' 

this does not sound like I want to promote this to a customer?

Reply:
@faase Good point. I say leave as is, i.e. negate the earlier +1.